src/containers/dashboard/innopoint-transfer.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import Card from 'ui/card.svelte';
+  import AutoComplete from '@/components/projects/new/autocomplete.svelte';
+  import SendMoney from '@/components/dashboard/send-money.svelte';
+  import SendComment from '@/components/dashboard/send-comment.svelte';
+  import SeeProfile from '@/components/dashboard/see-profile.svelte';
+  import options from '@/constants/autocomplete-options';
+  let users = [];
+</script>
+
+<Card classname="card individual-ctls">
+  <div class="title">
+    <svg src="/images/icons/user.svg" class="icon" />
+    Individual user controls
+  </div>
+  <div class="panel">
+    <div class="user-selection">
+      <AutoComplete bind:value={users} {options} />
```

Comment:
To fix the `users` being array of strings problem, you may add props such as `item-text`, `item-value`, and `return-object` to pass an object and specify which property to use for the text and for the emitted value, or to return the whole object instead of just one field. It can also be used as an array of strings without specifying any of that.
This is the approach taken by vuetify.

Reply:
Yeah, a lot refactor will be performed further